tests/stdlib/tarithmetics.nim

```diff
@@ -0,0 +1,44 @@
+discard """
+  targets: "c cpp js"
+"""
+
+# TODO: in future work move existing arithmetic tests here, where they belong: Nim/tests/arithm
```

Comment:
```suggestion
# TODO: in future work move existing arithmetic tests here, where they belong
```
(tests/arithm doesn't exist; the point of that comment was to move existing tests (and add new tests) related to procs defined in lib/system/arithmetics.nim to this test file